components/core/src/os/filesystem/windows.rs

```diff
@@ -25,7 +26,11 @@ pub fn chmod(r_path: *const c_char, mode: u32) -> c_int {
         let path = CStr::from_ptr(r_path).to_str().unwrap();
         match Path::new(path).exists() {
             false => 1,
-            true => 0
-        } 
+            true => 0,
+        }
     }
 }
+
+pub fn symlink<P: AsRef<Path>, Q: AsRef<Path>>(src: P, dst: Q) -> io::Result<()> {
+    unimplemented!();
```

Comment:
Junctions can cross drive boundaries and are the only link type on windows that can do directory links.  Hard links have to be on the same physical and logical volume and can only reference individual files.  

 https://msdn.microsoft.com/en-us/library/windows/desktop/aa365006(v=vs.85).aspx